lib/std/cli.zig

```diff
@@ -136,13 +149,21 @@ test parse {
             level: i8 = -1,
             /// Parsed as the name of the member `--color=never`.
             color: enum { auto, never, always } = .auto,
-            /// --seed=0x<something> is actually passed in by the `zig test` system (as of 0.14.1), which we receive here.
+
+            // The below parameters are actually passed into the `zig test` process,
+            // so we have to receive them here (as of zig 0.15.1).
             seed: u32 = 0,
             @"cache-dir": []const u8 = "",
             listen: []const u8 = "",
         },
-        /// Receives the rest of the arguments.
-        positional: []const [:0]const u8 = &.{},
+        positional: struct {
+            /// First positional (non-named) argument:
+            input: [:0]const u8 = "",
+            /// Second positional argument is declared as optional:
+            reptitions: u32 = 1,
```

Comment:
Potential typo of "repetitions"? (completely inconsequential)

Reply:
5a11f4c1b61ea6f3c546c58c957b3479bd38e422 didn't really fix the typo. Now it's "rep**i**titions"

Reply:
:facepalm: :joy: fixed again (in the next commit). thanks yall.